app/services/identifier_service.rb

```diff
@@ -1,16 +1,23 @@
 # frozen_string_literal: true
 class IdentifierService
   def self.mint_or_update(resource:)
-    if resource.identifier.present?
+    if identifier_for(resource).present?
```

Comment:
Something for the future - maybe we should look into making this an instance now, if we're calling private class methods?